Gemfile

```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 ruby '2.2.2' # this should match `.ruby-version` and docs/setup.md
 
+gem 'about_yml'
```

Comment:
Take another look:

https://github.com/18F/about_yml/#installation

Maybe they expanded that section since you looked at it? Totally fine with including it (in `development` and `test` only, with `require: false`) if you want to run it as part of CI.